### DISCUSSION TITLE
Help please 3

### DISCUSSION THREAD

**Discussion by IvanTymoshchuk** [IvanTymoshchuk/test-work#5] [Q&A] [answered]
Функція checkForSpam(message) приймає рядок (параметр message), перевіряє його на вміст заборонених слів spam і sale, і повертає результат перевірки. Слова в рядку параметра message можуть бути у довільному регістрі, наприклад SPAM або sAlE.

Якщо знайшли заборонене слово (spam або sale) то функція повертає буль true.
Якщо в рядку відсутні заборонені слова, функція повертає буль false.

**Comment by MargoMarm** [comment, marked as answer]
function checkForSpam(message) {
  let result;
  // Change code below this line
message.toLowerCase().includes("spam") || message.toLowerCase().includes("sale") ? result=true : result=false ;
  // Change code above this line
  return result;
}
